examples/tilted_bottom_boundary_layer.jl

```diff
@@ -98,10 +98,17 @@ coriolis = ConstantCartesianCoriolis(f = 1e-4, rotation_axis = ĝ)
 N² = 1e-5 # s⁻² # background vertical buoyancy gradient
 B_field = BackgroundField(constant_stratification, parameters=(; ĝ, N² = N²))
 
-# Because the bottom boundary condition is that there must be zero *total* diffusive
-# flux across the seafloor, the default `FluxBoundaryCondition()` is insufficient;
-# instead, the boundary condition on the perturbation flux must take the background
-# into account,
+# We choose to impose a bottom boundary condition of zero *total* diffusive buoyancy
+# flux across the seafloor,
+# ```math
+# ∂_z B = ∂_z b + N^{2} \cos{\theta} = 0.
```

Comment:
Beautiful! Sorry to be a nag (and correct me if I am wrong) but do we not use `B` to denote the background field in this example? Let's just make sure to use consistent notation in the code and the math. But please feel free to make the notation whatever you like of course!

Reply:
and to clarify this we could write something like

```
where ``∂_z B = N^{2} \cos{\theta}`` is the vertical buoyancy gradient of the background buoyancy field.
```

Reply:
Ah, but it's called `B_field` above. That should be `B∞_field`?